tiledb/bioimg/helpers.py

```diff
@@ -17,8 +18,11 @@ class ReadWriteGroup:
     def __init__(self, uri: str):
         parsed_uri = urlparse(uri)
         # normalize uri if it's a local path (e.g. ../..foo/bar)
-        if parsed_uri.scheme in ("", "file"):
-            uri = str(Path(parsed_uri.path).resolve())
+
+        # Windows paths produce single letter scheme matching the drive letter
+        # Unix absolute path produce an empty scheme
+        if len(parsed_uri.scheme) < 2 or parsed_uri.scheme == "file":
```

Comment:
Generally magic numbers should be avoided, but since we will revisit the whole path resolution in future ticket lets leave it as is.